python/function.py

```diff
@@ -532,32 +537,61 @@ def count(self, value):
 
 	@classmethod
 	def undetermined(self):
+		"""
+		Create a PossibleValueSet object of type UndeterminedValue.
+
+		:return: PossibleValueSet object of type UndeterminedValue
+		:rtype: PossibleValueSet
+		"""
 		return PossibleValueSet()
 
 	@classmethod
 	def constant(self, value):
-		""" """
+		""" 
+		Create a constant valued PossibleValueSet object.
+
+		:param int value: Integer value of the constant
+		:rtype: PossibleValueSet
+		"""
 		result = PossibleValueSet()
 		result.type = RegisterValueType.ConstantValue
 		result.value = value
 		return result
 
 	@classmethod
 	def constant_ptr(self, value):
+		""" 
+		Create constant pointer valued PossibleValueSet object.
+
+		:param int value: Integer value of the constant pointer
+		:rtype: PossibleValueSet
+		"""
 		result = PossibleValueSet()
 		result.type = RegisterValueType.ConstantPointerValue
 		result.value = value
 		return result
 
 	@classmethod
 	def stack_frame_offset(self, offset):
+		""" 
+		Create a PossibleValueSet object for a stack frame offset.
+
+		:param int value: Integer value of the offset
+		:rtype: PossibleValueSet
+		"""
 		result = PossibleValueSet()
 		result.type = RegisterValueType.StackFrameOffset
 		result.value = value
 		return result
 
 	@classmethod
 	def signed_range_value(self, ranges):
+		"""
+		Create a PossibleValueSet object for a signed range of values.
+
+		:param list(ValueRange) ranges: List of ValueRanges
```

Comment:
An example here would be nice since it's the only type not immediately obvious from the existing context.